var/spack/repos/builtin/packages/py-ytopt/package.py

```diff
@@ -15,12 +15,16 @@ class PyYtopt(PythonPackage):
     homepage = "https://github.com/ytopt-team/ytopt"
     url      = "https://github.com/ytopt-team/ytopt/archive/refs/tags/v0.0.1.tar.gz"
 
+    version('0.0.3', sha256='eac6ab87d4fd27517f136880016359c5b24836ec009e8cc9b4073a6c5edb17af')
     version('0.0.2', sha256='5a624aa678b976ff6ef867610bafcb0dfd5c8af0d880138ca5d56d3f776e6d71')
     version('0.0.1', sha256='3ca616922c8e76e73f695a5ddea5dd91b0103eada726185f008343cc5cbd7744')
 
+    variant('online', default=False, description='Install requirements for online tuning.')
```

Comment:
Can we make his variant start at v0.0.3? That means adding `when='@0.0.3:'` to the variant directive.

Reply:
Yes, that makes sense. I made the change. Thank you.